### PR TITLE
[TEVA-2997] Remove day from dates in employment section

### PR DESCRIPTION
**Pull request by cesidio**
## Jira ticket URL
https://dfedigital.atlassian.net/browse/TEVA-2997

## Screenshots of UI changes:
Screenshot 2021-09-01 at 16 16 03
Screenshot 2021-09-01 at 16 16 12
Screenshot 2021-09-01 at 16 16 31

### CODE REVIEW

**Review by alexbowen — APPROVED** [review]
have given the review app a try and seems fine

File: app/controllers/jobseekers/job_applications/breaks_controller.rb [hunk 1]
```diff
@@ -37,6 +37,7 @@ def employment_break
   def employment_break_params
     params.require(:jobseekers_job_application_details_break_form)
           .permit(:reason_for_break, :started_on, :ended_on)
+          .merge("started_on(3i)" => "1", "ended_on(3i)" => "1")
```
Comment (alexbowen) [inline app/controllers/jobseekers/job_applications/breaks_controller.rb:40]: just a question - what is this this `3i` business?

Comment (cesidio) [reply app/controllers/jobseekers/job_applications/breaks_controller.rb:40]: 1i is the year, 2i is the month and 3i is the day. What we do here is assign the first day of the month to the date. You can check more here:
https://govuk-form-builder.netlify.app/form-elements/date-field/

File: app/controllers/jobseekers/job_applications/employments_controller.rb [hunk 2]
```diff
@@ -37,6 +37,7 @@ def employment
   def employment_params
     params.require(:jobseekers_job_application_details_employment_form)
           .permit(:organisation, :job_title, :salary, :subjects, :main_duties, :started_on, :current_role, :ended_on)
+          .merge("started_on(3i)" => "1", "ended_on(3i)" => "1")
```
[the remark this answers is not in the document]
Comment (cesidio) [reply app/controllers/jobseekers/job_applications/employments_controller.rb:40]: I have tried but it does not work if I do not add the day

**Pull request merged**